### ISSUE TITLE
Discord rpc not connecting on arch linux

### ISSUE THREAD

**Issue by legendlife** [leonardssh/vscord#171]
I'm using vanilla arch

I installed discord and visual studio code with discord's arch user repo.
The rpc doesn't show on discord.

**Comment by xhayper** [comment]
Can you please send the log?

**Comment by legendlife** [comment]
> Can you please send the log?

what should I type?

**Comment by xhayper** [comment]
you can open it through output tab

**Comment by xhayper** [comment]
![image](https://user-images.githubusercontent.com/46065349/213907304-f64ffe64-b562-4c2b-8be2-dd27a1319ab4.png)

**Comment by legendlife** [comment]
[6:14:36 pm] [INFO] Discord Rich Presence for VS Code activated.
[6:14:36 pm] [INFO] Registered Discord Rich Presence commands
[6:14:37 pm] [INFO] Successfully connected to Discord
[6:14:37 pm] [INFO] [004] Debug: Enabling Discord Rich Presence
[6:14:37 pm] [INFO] [004] Debug: Logging in with client ID "782685898163617802"
[6:14:37 pm] [INFO] [004] Debug: Login - isConnected true isReady 782685898163617802
[6:14:37 pm] [INFO] [004] Debug: login - [object Object]
[6:14:37 pm] [INFO] [004] Debug: Client Should be logged in
[6:14:37 pm] [INFO] [004] Debug: enable - [object Object]
[6:14:37 pm] [INFO] [004] Debug: Enabled - isConnected true isReady 782685898163617802
[6:14:37 pm] [INFO] Successfully logged in to Discord with client ID 782685898163617802

**Comment by legendlife** [comment]
![image](https://user-images.githubusercontent.com/80240062/214833504-9c330b1a-908c-47c7-a303-3c7b716577f8.png)

it says it's connected but as you can see the screenshot

**Comment by xhayper** [comment]
TLDR;

Check if discord activity is enabled
